code/modules/reagents/machinery/chem_master.dm

```diff
@@ -582,8 +601,14 @@ var/global/list/pillIcon2Name = list("oblong purple-pink", "oblong green-white",
 			//
 			// PILL ICONS
 			//
+
+			dat += "<HR><a href='?src=\ref[src];pill_icon_toggle=1'>Toggle Additional Pill Icons</a>"
```

Comment:
You're adding an unnecessary horizontal line, try adding this inside the generate_pill_icon_div() proc